xbmc/Application.cpp

```diff
@@ -3501,6 +3501,8 @@ void CApplication::OnPlayBackEnded()
   CDarwinUtils::EnableOSScreenSaver(true);
 #endif
 
+  CServiceBroker::GetPVRManager().OnPlaybackEnded(m_itemCurrentFile);
```

Comment:
the other direction that PVR updates currentFileItem in application should not be required anymore: https://github.com/xbmc/xbmc/search?utf8=%E2%9C%93&q=setcurrentfileitem&type=